server/pse/pse_windows.go

```diff
@@ -255,7 +256,12 @@ func ProcUsage(pcpu *float64, rss, vss *int64) error {
 		return fmt.Errorf("could not find pid in performance counter results")
 	}
 	// assign values from the performance counters
-	*pcpu = cpuAry[idx]
+	cpu := cpuAry[idx]
+	if cpu > 0.0 {
+		cpu /= float64(runtime.NumCPU())
```

Comment:
That being said, checking NumCPU, this is already done at the low level, so don't bother.